src/idiomatic/foundations-api-design/predictable-api/naming-conventions/06-with.md

```diff
@@ -0,0 +1,31 @@
+---
+minutes: 5
+---
+
+# With
+
+Prefix for builder-style configuration.
```

Comment:
I'd avoid using the word "builder" because this usage pattern is significantly different. A builder requires calling some method to finalize object construction (`.build();` at the end), while `with` methods in Rust return the updated value directly.